composer.json

```diff
@@ -71,6 +71,7 @@
         "drupal/webform_validation": "^2.0",
         "drupal/webform_views": "^5.0@alpha",
         "drupal/workflow_participants": "^2.4",
+        "fig/http-message-util": "^1.1",
```

Comment:
i see this is only needed for getting HTTP methods names as consts?
if so, i suggest using `Symfony\Component\HttpFoundation\Request` for that, it provides the methods names in the same way and does not need to be requested separately - already part of Drupal.

Reply:
Fixed